jupyter-extension/jupyterlab_pachyderm/tests/test_handlers.py

```diff
@@ -536,3 +536,13 @@ async def test_health(mock_client, jp_fetch):
     mock_client.health.return_value = json.dumps({"status": "running"})
     r = await jp_fetch(f"/{NAMESPACE}/{VERSION}/health")
     assert json.loads(r.body) == {"status": "running"}
+
+
+@pytest.mark.skipif(sys.version_info < (3, 7), reason="requires python3.7 or higher")
+async def test_pps_get(jp_fetch):
+    response = await jp_fetch(f"/{NAMESPACE}/{VERSION}/pps/_create/Untitled.ipynb")
```

Comment:
Where is the `Untitled.ipynb` notebook?

Reply:
It's not a real file. I'll change its name to make this more clear